runtime/src/main/scala/akka/grpc/javadsl/GrpcExceptionHandler.scala

```diff
@@ -32,7 +33,9 @@ object GrpcExceptionHandler {
 
   /** INTERNAL API */
   @InternalApi
-  private def default(system: ActorSystem): jFunction[Throwable, Trailers] =
+  private def default(system: ActorSystem): jFunction[Throwable, Trailers] = {
+    val log = Logging(system, "akka.grpc.javadsl.GrpcExceptionHandler")
+
```

Comment:
This isn't called repeatedly, which would add overhead of creating logger?

Reply:
In the Kalix proxy we create the partial function once at bootstrap (Serve.createRoute) and re-use for all calls, so a slight optimization for the peer closed and unexpected error scenarios. Not very important though.